### ISSUE TITLE
Bar chart does not show correct total number of screened records

### ISSUE THREAD

**Issue by WordliczekL** [asreview/asreview#1965]
Hello,
Seems to have a similar issue with visualisation.
Here's a descriptive summary of my project details:
![obraz](https://github.com/user-attachments/assets/39f034ce-ff85-4fa0-932b-db2ef63c61ba)
...and here's accompanying ROC curve:
![obraz](https://github.com/user-attachments/assets/ab5297d5-3fda-40f0-9c64-5d986728288d)
Two questions seem to be clear: 
1. why the chart chunks off at "13" while the number of relevant records is 18?
2. why the number of reviewed records is 137 in the chart, while the actual value is 152?
Prior knowledge was 5 relevant and 10 irrelevant records. This would, I guess, resolve question no. 1 (am I correct?) but not quite question no. 2.
Is this a kind of a glitch, or rather, more likely, I did something wrong?
Appreciate any assistance! :))
 

### Discussed in https://github.com/asreview/asreview/discussions/1902

<div type='discussions-op-text'>

<sup>Originally posted by **RiannevanderFeen** October 23, 2024</sup>
Hello,

I had hoped to use the figures from the analytics in my paper, but I noticed that it does not show the correct total amount of screened records. 

I screened >10% in phase 2 (2738), but the figure (the bar chart!) states 2462 as you can see below:

![image](https://github.com/user-attachments/assets/25c1679d-9224-4212-94fb-57ae5589efc2)

</div>

**Comment by BerkeYazan** [comment]
Hello!

For the general analytics section, we added a [switch](https://github.com/asreview/asreview/pull/1766) to change between showing or hiding the prior knowledge. So in this case, `Total Records` would switch between 137 (hiding prior knowledge) and 152 (showing prior knowledge). 

This is the case for the general analytics. For the graphs, however, it's a bit more tricky. We opted to exclude the prior knowledge because of two reasons:

1. The graphs have a time axis, so, they are only meaningful if the labelling order is preserved. In most cases, prior knowledge doesn't come with chronology data, making the visualization inaccurate and confusing.
2. From our point of view, the analytics section is a way to understand your progress **while using ASReview**, as it reflects the performance of the models and the screener. Even though the prior knowledge labelling can also be done inside ASReview before starting the actual process, they belong to a different part of active learning. 

**So,** the exclusion of prior knowledge would explain 2 of the 3 differences in your case. The differences between 18 - 13 and 152 - 137 are because of this.

**However,** this doesn't explain the 2738 - 2462 difference in the bar graph. Is it possible that your dataset had prior knowledge before you started screening? If not, it might be a bug with the axis values of the graph, I'll take a look. 

Let us know if you think it's integral to see the prior knowledge in the graphs as well, we can try to come up with a clever solution.
